service/src/main/java/com/ge/predix/acs/privilege/management/dao/ResourceHierarchicalRepository.java

```diff
@@ -4,7 +4,7 @@
 
 public interface ResourceHierarchicalRepository {
 
-    ResourceEntity getByZoneAndResourceIdentifierWithInheritedAttributes(final ZoneEntity zone,
+    ResourceEntity getInheritedAttributes(final ZoneEntity zone,
             final String resourceIdentifier);
 
 }
```

Comment:
@irinaepshteyn 
@sanjeevchopra 
The intent of "getByZoneAndResourceIdentifierWithInheritedAttributes" was to follow the Spring Data JAP convention. However, since it's not really Spring Data I suppose we can follow any naming conventions.

"getInheritedAttributes" is a bit misleading in that it implies that it is returning something else than an actual instance of a resource object.

"getResourceWithInheritedAttributes" is somewhat redundant in that the method signature is clear about what the return type is.

"getWithResourceAttributes" would probably be the more canonical naming convention but is confusing from an English language point of view.

"getLost" is likely what you're both thinking at this point.

I think being more explicit is the lesser of two evils. I would say let's go with "getResourceWithInheritedAttributes".